docs/02-get-started/04-trigger-workload-with-event.md

```diff
@@ -112,22 +111,15 @@ All the published events of this type are then forwarded to an HTTP endpoint cal
 Run:
 ```bash
 cat <<EOF | kubectl apply -f -
-   apiVersion: eventing.kyma-project.io/v1alpha1
+   apiVersion: eventing.kyma-project.io/v1alpha2
    kind: Subscription
    metadata:
      name: lastorder-sub
      namespace: default
    spec:
-     filter:
-       filters:
-       - eventSource:
-           property: source
-           type: exact
-           value: ""
-         eventType:
-           property: type
-           type: exact
-           value: sap.kyma.custom.myapp.order.received.v1
+     source: myapp
+     types:
+       - order.received.v1
```

Comment:
(nit)
perhaps it's also good to explicitly set the `typeMatching` here, b/c it is defaulting to `exact`.

Reply:
is it? isn't it by default set to `standard`?

Reply:
yes. sorry for the confusion, I meant the standard is default. but can we still add the `standard` explicitly ?

Reply:
IMO, lets keep it simple. `Exact` is anyway for advance users and mentioning `standard` here would not benefit the novice user anyway.

Reply:
Ok, let's keep it simple.